roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -7,7 +7,7 @@ server {
   listen 80;
   {% endif %}
 
-  server_name  {{ item.value.site_hosts | join(' ') }};
+  server_name  {% for host in item.value.site_hosts %} {{ host }} {% if item.value.multisite.subdomains %} *.{{ host }} {% endif %} {% endfor %};
```

Comment:
@austinpray this seems the most logical way to handle it. 

> I'm told this fails 

@fullyint  have you been told why it fails?
I'm not sure if I understand [this](https://github.com/phinze/landrush#wildcard-subdomains) correctly (haven't tested), but to me it looks like using:

```
site_hosts:
  - site1.com
```

`*.site.com` would automatically be routed correctly to the IP of the vagrant box?